src/index.js

```diff
@@ -45,8 +45,7 @@ class FontAwesome extends React.Component {
       stack,
       tag = 'span',
       ariaLabel,
-      ...props,
-    } = this.props
+      ...props } = this.props
```

Comment:
Ok, fyi I did a little googling and apparently the spec says that in this case the the trailing comma should not be enforced. Anyway I will revert the change to the original state.